src/AuthService/Application/Core/Domain/Entities/Base/IEntity.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Domain.Entities.Base;
+
+/// <summary>
+/// The <c>IEntity</c> interface represents a base entity with unique identifier
+/// </summary>
+public interface IEntity
```

Comment:
I would suggest using a base abstract class here and not the interface because of the nature and sense behind interfaces and abstract classes. See this article https://enterprisecraftsmanship.com/posts/entity-base-class/ as example